### PR TITLE
fix/Deleting repo with issues

### PR DESCRIPTION
**Pull request by a-kuharenko**

### CODE REVIEW

File: server/data/migrations/20190822201536-update-issue-repositoryId-column.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.sequelize.transaction((/* transaction */) => Promise.all([
+    queryInterface.removeColumn('issues', 'repositoryId'),
+    queryInterface.addColumn('issues', 'repositoryId', {
```
Comment (OlgaGnatenko) [inline server/data/migrations/20190822201536-update-issue-repositoryId-column.js]: Theoretically, you can update the existing field with addConstraint as described here: https://github.com/sequelize/sequelize/pull/7108

Comment (a-kuharenko) [reply server/data/migrations/20190822201536-update-issue-repositoryId-column.js]: I tried, but it is not working in my case

Comment (OlgaGnatenko) [reply server/data/migrations/20190822201536-update-issue-repositoryId-column.js]: Understood, it really looks still experimental in some cases

**Comment by a-kuharenko** [comment]
No, it is everywhere. This migration will fix deleting from DB

**Pull request merged**